sphinx/util/inspect.py

```diff
@@ -108,6 +108,10 @@ def safe_getattr(obj, name, *defargs):
     try:
         return getattr(obj, name, *defargs)
     except Exception:
+        # for NotImplementedError mixed abstractproperty, let's check for the
+        # property name before raising an error
```

Comment:
This comment is not suitable. This also handles the exceptions other than NotImplementedError. abstractproperty also.